etc/config/openclc.amazon.properties

```diff
@@ -0,0 +1,189 @@
+compilers=&armoclcclang32:&armoclcclang64
+defaultCompiler=armoclcclang64
+demangler=/opt/compiler-explorer/gcc-11.1.0/bin/c++filt
+objdumper=/opt/compiler-explorer/gcc-11.1.0/bin/objdump
+needsMulti=false
+supportsBinary=false
+
+# Clang for Arm
+# Provides 32- and 64-bit menu items for clang-9, clang-10, clang-11 and trunk
+group.armoclcclang32.groupName=Arm 32-bit clang
+group.armoclcclang32.compilers=armv7-oclcclang900:armv7-oclcclang901:armv7-oclcclang1000:armv7-oclcclang1001:armv7-oclcclang1100:armv7-oclcclang1101:armv7-oclcclang-trunk
+group.armoclcclang32.isSemVer=true
+group.armoclcclang32.compilerType=clang
+group.armoclcclang32.supportsExecute=false
+group.armoclcclang32.instructionSet=arm32
+# The -Dkernel= -D__kernel= workaround is required to prevent the Clang crash reported in http://llvm.org/PR50841
+group.armoclcclang32.baseOptions=-Xclang -finclude-default-header -Dkernel= -D__kernel=
```

Comment:
stating from clang 13 we don't need to pass `-Xclang -finclude-default-header` although it still works with it too it makes clang super slow. Will it be possible to add it conditionally based on the release version?

Reply:
Yes, I can do this - thanks!